node-module-system/calculate.js

```diff
@@ -0,0 +1,15 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+process.argv.forEach((val, index) => {
```

Comment:
Looping over the args is somewhat unusual. You know exactly which argv should contain the operator, so maybe there's no reason to have a loop?